packages/spritesheet/src/Spritesheet.ts

```diff
@@ -3,6 +3,41 @@ import { Texture, BaseTexture } from '@pixi/core';
 import { getResolutionOfUrl } from '@pixi/utils';
 import type { Dict } from '@pixi/utils';
 import type { resources } from '@pixi/core';
+import type { IPointData } from '@pixi/math';
+
+/**
+ * Represents the JSON data for a spritesheet atlas.
+ */
+export interface ISpritesheetFrameData {
+    frame: {
+        x: number;
+        y: number;
+        w: number;
+        h: number;
+    };
+    trimmed?: boolean;
+    rotated?: boolean;
+    sourceSize?: {
+        w: number;
+        h: number;
+    };
+    spriteSourceSize?: {
+        x: number;
+        y: number;
+    };
+    anchor?: IPointData;
+}
+
+/**
+ * Atlas format.
+ */
+export interface ISpritesheetData {
+    frames: Dict<ISpritesheetFrameData>;
+    animations: Dict<string[]>;
```

Comment:
Should animations be optional? It seems like a more advanced thing.

Reply:
Good call @andrewstart, missed that one.